rhel/ubi9/Dockerfile

```
@@ -0,0 +1,112 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1214 as jre-build
```

A newer version was released recently.  Once this is merged, the automated updates should take care of it, but probably best to use most recent release from the start.
```suggestion
FROM registry.access.redhat.com/ubi9/ubi:9.4-1214.1726694543 as jre-build
```